src/CreeDictionary/API/search/types.py

```diff
@@ -4,85 +4,18 @@
 import json
 from dataclasses import dataclass, field
 from enum import Enum
-from typing import NewType, Iterable, Tuple, Optional, cast
-
-from typing_extensions import Protocol
+from typing import NewType, Optional
 
 from morphodict.lexicon.models import Wordform, wordform_cache
-from CreeDictionary.API.schema import SerializedLinguisticTag
 from CreeDictionary.API.search import ranking
-from CreeDictionary.CreeDictionary.relabelling import read_labels
-from CreeDictionary.utils.types import FSTTag, Label
+
 
 Preverb = Wordform
 Lemma = NewType("Lemma", Wordform)
 MatchedEnglish = NewType("MatchedEnglish", str)
 InternalForm = NewType("InternalForm", str)
 
 
-class LinguisticTag(Protocol):
```

Comment:
This protocol and its implementors were used by `relevant_tags`, which is no longer used. Which is a good thing because these classes are super confusing!